src/scripts/firebase_structure.js

```diff
@@ -0,0 +1,106 @@
+export const produto = [
+    {
+        categoria:[
+            {
+                descrição: "",
+                nome: ""
+            }
+        ]
+    },
+    {
+        item:[
+            {
+                dataCompra: "",
+                dataValidade: "",
+                fornecedorToken: "",
+                preço: "",
+                quantidade: 0
+            }
+
+        ]
+  },
+  {
+        marca:[
+            {
+                nome: ""
+            }
+        ]
+  },
+  {
+      unidadeMedida:[
+          {
+              abreviação: "",
+              descrição: ""
+          }         
+      ]
+
+  }
```

Comment:
Algumas virgulas estão faltando.
```suggestion
  },
```